test/bigquery.test.ts

```diff
@@ -301,4 +250,677 @@ describe('BigQueryFormatter', () => {
+          CLONE mydataset.mytable`;
+        const expected = dedent`
+          ${createSnap}
+            mydataset.mytablesnapshot
+          CLONE
+            mydataset.mytable`;
+
+        expect(format(input)).toBe(expected);
+      });
+    });
+
+    const createViewVariations = [
+      'CREATE VIEW',
+      'CREATE OR REPLACE VIEW',
+      'CREATE VIEW IF NOT EXISTS',
+    ];
+    const createMaterializedView = [
+      'CREATE MATERIALIZED VIEW',
+      'CREATE OR REPLACE MATERIALIZED VIEW',
+      'CREATE MATERIALIZED VIEW IF NOT EXISTS',
+    ];
+
+    createViewVariations.concat(createMaterializedView).forEach(createView => {
+      it(`Supports ${createView}`, () => {
+        const input = `
+          ${createView} my_dataset.my_view AS (
+          SELECT t1.col1, t1.col2 FROM my_dataset.my_table)`;
+        const expected = dedent`
+          ${createView}
+            my_dataset.my_view AS (
+              SELECT
+                t1.col1,
+                t1.col2
+              FROM
+                my_dataset.my_table
+            )`;
+        expect(format(input)).toBe(expected);
+      });
+    });
+
+    const createExternalTableVariations = [
+      'CREATE EXTERNAL TABLE',
+      'CREATE OR REPLACE EXTERNAL TABLE',
+      'CREATE EXTERNAL TABLE IF NOT EXISTS',
+    ];
+    createExternalTableVariations.forEach(createTable => {
+      it(`Supports ${createTable}`, () => {
+        const input = `
+          ${createTable} dataset.CsvTable
+          WITH PARTITION COLUMNS (
+            field_1 STRING,
+            field_2 INT64
+          )
+          OPTIONS(
+            format = 'CSV',
+            uris = ['gs://bucket/path1.csv']
+          )`;
+        const expected = dedent`
+          ${createTable}
+            dataset.CsvTable
+          WITH PARTITION COLUMNS
+            (field_1 STRING, field_2 INT64) OPTIONS(format = 'CSV', uris = ['gs://bucket/path1.csv'])`;
+        expect(format(input)).toBe(expected);
+      });
+    });
+
+    const createFunctions = [
+      'CREATE FUNCTION',
+      'CREATE OR REPLACE FUNCTION',
+      'CREATE FUNCTION IF NOT EXISTS',
+    ];
+    const createTempFunctions = [
+      'CREATE TEMP FUNCTION',
+      'CREATE OR REPLACE TEMP FUNCTION',
+      'CREATE TEMP FUNCTION IF NOT EXISTS',
+      'CREATE TEMPORARY FUNCTION',
+      'CREATE OR REPLACE TEMPORARY FUNCTION',
+      'CREATE TEMPORARY FUNCTION IF NOT EXISTS',
+    ];
+
+    createFunctions.concat(createTempFunctions).forEach(createFunction => {
+      it(`Supports ${createFunction}`, () => {
+        const input = `
+          ${createFunction} mydataset.myFunc(x FLOAT64, y FLOAT64)
+          RETURNS FLOAT64
+          AS (x * y);`;
+        const expected = dedent`
+          ${createFunction}
+            mydataset.myFunc (x FLOAT64, y FLOAT64) RETURNS FLOAT64 AS (x * y);`;
+        expect(format(input)).toBe(expected);
+        expect(format(input, { aliasAs: 'always' })).toBe(expected);
```

Comment:
Huh, huh... finally figured out that this got fixed by the addition of `RETURNS` to list of keywords.

Should at least add a comment here mentioning that it's a regression test for issue #309